### ISSUE TITLE
Large objects are not converted.

### ISSUE THREAD

**Issue by makarhuyaker** [Lewinator56/swMesh2XML_repo#5]
I wanted to convert a hangar to .mesh, but it gave me an error, I reduced the hangar to 0.1 in a blender, but I still get an error here is a screenshot.
![Снимок экрана (1136)](https://user-images.githubusercontent.com/63492966/96494486-17721180-124f-11eb-9137-d1f8d001d6a2.png)

**Comment by Lewinator56** [comment]
there is a vertex limit of 21845, if you exceeded this limit, this will happen, however, i can look into it if you send me the mesh, potentially you have just missed a step in setting it up for the converter

**Comment by Dogboy394** [comment]
> there is a vertex limit of 21845, if you exceeded this limit, this will happen, however, i can look into it if you send me the mesh, potentially you have just missed a step in setting it up for the converter

i have the same problem

**Comment by Lewinator56** [comment]
the model has too many vertices

**Comment by Dogboy394** [comment]
yeah ik and i figured it out its all good

**Comment by Dogboy394** [comment]
i imported a truck a f450

**Comment by Dogboy394** [comment]
> the model has too many vertices

can you make it so that there isnt a limit

**Comment by Lewinator56** [comment]
> > the model has too many vertices
> 
> can you make it so that there isnt a limit

no, im afraid not, its a limitation of the engine used by stormworks

**Comment by Dogboy394** [comment]
> > > the model has too many vertices
> > 
> > 
> > can you make it so that there isnt a limit
> 
> no, im afraid not, its a limitation of the engine used by stormworks

What would happen if there was no limit

**Comment by Lewinator56** [comment]
> > > > the model has too many vertices
> > > 
> > > 
> > > can you make it so that there isnt a limit
> > 
> > 
> > no, im afraid not, its a limitation of the engine used by stormworks
> 
> What would happen if there was no limit

its not technically possible, the engine uses 16 bit unsigned integers to define vertices in the mesh, that is; 65535 vertices. As a triangle requires 3 vertices, you can only have 1/3 the number of triangles as vertices in a mesh. your model simply exceeds one of these limits. A more complex engine may use 32 or 64 bit uints, both of which offer considerably higher vertex counts.
